<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="3" passed="0" failed="3" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2025-09-26T11:58:13 IST" name="Default suite" finished-at="2025-09-26T11:58:30 IST" duration-ms="16819">
    <groups>
    </groups>
    <test started-at="2025-09-26T11:58:13 IST" name="Default test" finished-at="2025-09-26T11:58:30 IST" duration-ms="16819">
      <class name="Tests.RegisterTest">
        <test-method is-config="true" signature="setUp()[pri:0, instance:Tests.RegisterTest@692f203f]" started-at="2025-09-26T11:58:13 IST" name="setUp" finished-at="2025-09-26T11:58:17 IST" duration-ms="4212" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="testUserRegistration(java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String)[pri:0, instance:Tests.RegisterTest@692f203f]" started-at="2025-09-26T11:58:17 IST" name="testUserRegistration" data-provider="registrationData" finished-at="2025-09-26T11:58:19 IST" duration-ms="2218" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[Ramamani]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Seshadri]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[123 Test Street]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[Bangalore]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[Karnataka]]>
              </value>
            </param>
            <param index="5">
              <value>
                <![CDATA[560001]]>
              </value>
            </param>
            <param index="6">
              <value>
                <![CDATA[9876543210]]>
              </value>
            </param>
            <param index="7">
              <value>
                <![CDATA[123456789]]>
              </value>
            </param>
            <param index="8">
              <value>
                <![CDATA[RamamaniQA1]]>
              </value>
            </param>
            <param index="9">
              <value>
                <![CDATA[Vaish@123]]>
              </value>
            </param>
            <param index="10">
              <value>
                <![CDATA[Vaish@123]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertTrue(Assert.java:57)
at org.testng.Assert.assertTrue(Assert.java:67)
at Tests.RegisterTest.testUserRegistration(RegisterTest.java:27)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:228)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:961)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:201)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:819)
at org.testng.TestRunner.run(TestRunner.java:619)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1228)
at org.testng.TestNG.runSuites(TestNG.java:1134)
at org.testng.TestNG.run(TestNG.java:1101)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testUserRegistration -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:Tests.RegisterTest@692f203f]" started-at="2025-09-26T11:58:19 IST" name="tearDown" finished-at="2025-09-26T11:58:19 IST" duration-ms="198" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="setUp()[pri:0, instance:Tests.RegisterTest@692f203f]" started-at="2025-09-26T11:58:19 IST" name="setUp" finished-at="2025-09-26T11:58:22 IST" duration-ms="2829" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="testUserRegistration(java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String)[pri:0, instance:Tests.RegisterTest@692f203f]" started-at="2025-09-26T11:58:22 IST" name="testUserRegistration" data-provider="registrationData" finished-at="2025-09-26T11:58:25 IST" duration-ms="2216" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[Lakshmi]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Narayan]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[456 Main Road]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[Chennai]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[Tamil Nadu]]>
              </value>
            </param>
            <param index="5">
              <value>
                <![CDATA[600001]]>
              </value>
            </param>
            <param index="6">
              <value>
                <![CDATA[9123456789]]>
              </value>
            </param>
            <param index="7">
              <value>
                <![CDATA[987654321]]>
              </value>
            </param>
            <param index="8">
              <value>
                <![CDATA[LakshmiQA2]]>
              </value>
            </param>
            <param index="9">
              <value>
                <![CDATA[Lakshmi@123]]>
              </value>
            </param>
            <param index="10">
              <value>
                <![CDATA[Lakshmi@123]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertTrue(Assert.java:57)
at org.testng.Assert.assertTrue(Assert.java:67)
at Tests.RegisterTest.testUserRegistration(RegisterTest.java:27)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:228)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:961)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:201)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:819)
at org.testng.TestRunner.run(TestRunner.java:619)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1228)
at org.testng.TestNG.runSuites(TestNG.java:1134)
at org.testng.TestNG.run(TestNG.java:1101)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testUserRegistration -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:Tests.RegisterTest@692f203f]" started-at="2025-09-26T11:58:25 IST" name="tearDown" finished-at="2025-09-26T11:58:25 IST" duration-ms="168" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="setUp()[pri:0, instance:Tests.RegisterTest@692f203f]" started-at="2025-09-26T11:58:25 IST" name="setUp" finished-at="2025-09-26T11:58:27 IST" duration-ms="2533" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="testUserRegistration(java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String)[pri:0, instance:Tests.RegisterTest@692f203f]" started-at="2025-09-26T11:58:27 IST" name="testUserRegistration" data-provider="registrationData" finished-at="2025-09-26T11:58:29 IST" duration-ms="2190" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[Anita]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Rao]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[789 Lake View]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[Hyderabad]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[Telangana]]>
              </value>
            </param>
            <param index="5">
              <value>
                <![CDATA[500001]]>
              </value>
            </param>
            <param index="6">
              <value>
                <![CDATA[9988776655]]>
              </value>
            </param>
            <param index="7">
              <value>
                <![CDATA[111222333]]>
              </value>
            </param>
            <param index="8">
              <value>
                <![CDATA[AnitaQA3]]>
              </value>
            </param>
            <param index="9">
              <value>
                <![CDATA[Anita@123]]>
              </value>
            </param>
            <param index="10">
              <value>
                <![CDATA[Anita@123]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertTrue(Assert.java:57)
at org.testng.Assert.assertTrue(Assert.java:67)
at Tests.RegisterTest.testUserRegistration(RegisterTest.java:27)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:228)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:961)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:201)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:819)
at org.testng.TestRunner.run(TestRunner.java:619)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1228)
at org.testng.TestNG.runSuites(TestNG.java:1134)
at org.testng.TestNG.run(TestNG.java:1101)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testUserRegistration -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:Tests.RegisterTest@692f203f]" started-at="2025-09-26T11:58:29 IST" name="tearDown" finished-at="2025-09-26T11:58:30 IST" duration-ms="178" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- Tests.RegisterTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
